src/NuGet.Core/NuGet.Packaging/Signing/ISignPackageReader.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace NuGet.Packaging.Signing
+{
+    /// <summary>
+    /// A readonly package that can provide signatures and a sign manifest from a package.
+    /// </summary>
+    public interface ISignPackageReader : IDisposable
```

Comment:
`ISignedPackageReader`?

Reply:
I'll rename things to use Signed instead